packages/web/src/scss/components/Timeline/_theme.scss

```diff
@@ -0,0 +1,24 @@
+@use '@tokens' as tokens;
+
+$timeline-max-width: 800px;
```

Comment:
I would expect that we would influence the maximum width using the Container/Grid component, but it probably depends on the design scenarios.